extras/rsyslog-container/rsyslog.conf

```diff
@@ -0,0 +1,26 @@
+module(load="imfile" PollingInterval="10")
+module(load="mmnormalize")
+input(type="imfile"
+      file="/var/log/glusterd2/glusterd2.log"
```

Comment:
This can be taken from env variable

Reply:
I experimented a bit by creating a container specific config file that contains the env variable(s) and then source this file when the rsyslog container starts. The env variables can then by accessed from within rsyslog.conf by following a particular syntax. But all this appears to be still in experimental stage as far as rsyslog is concerned as during testing, the rsyslog container did not come up properly. I think it would be better to wait until this feature of rsyslog is stable and keep things as is for now. I will keep on the lookout for this.